File: Selenium/Activities/Activity18.java
package Activity;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.Select;

import java.util.List;

public class Activity18 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/selenium/selects");
        System.out.println("Page Title :- " + driver.getTitle());
        Select sel=new Select(driver.findElement(By.id("multi-select")));
        sel.selectByVisibleText("Javascript");
        sel.selectByIndex(4);
        sel.selectByIndex(5);
        sel.selectByIndex(6);
        sel.selectByValue("node");
        sel.deselectByIndex(5);


        driver.close();

    }
}

File: Selenium/Activities/Activity18.py
from selenium import webdriver
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.firefox.service import Service as fireFoxService
from selenium.webdriver.common.by import By
import time
from selenium.webdriver.support.select import Select

service=fireFoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service)as driver:
    driver.get("https://training-support.net/selenium/selects")
    print("Page Title :- " + driver.title)
    sel=Select(driver.find_element(By.ID,"multi-select"))
    sel.select_by_visible_text("Javascript")
    sel.select_by_index(4)
    sel.select_by_index(5)
    sel.select_by_index(6)
    sel.select_by_value("node")
    sel.deselect_by_index(5)
    

    driver.close()